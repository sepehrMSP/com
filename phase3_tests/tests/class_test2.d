class c{
    void f(int a,int b){
        Print(a,b);
    }
    void g(int a,int b){
        Print(a);
        Print(b);
    }
}

class d extends c{
    void f(int a,int b){
        Print(a);
        Print(2.3);
        Print(b);
    }
}

int main(){
    c obj;
    obj=new d;
    obj.f(5,4);
}