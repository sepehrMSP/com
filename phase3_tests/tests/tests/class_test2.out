5
2.29999995
4
